python/tk_framework_desktopserver/certificates.py

```diff
@@ -97,12 +97,32 @@ def _check_call(self, ctx, cmd):
         :param cmd: Command to run.
 
         :raises CertificateRegistrationError: Raised when the subprocess doesn't return 0.
+
+        :returns: Command output.
         """
-        p = subprocess.Popen(cmd, stderr=subprocess.STDOUT, stdout=subprocess.PIPE, shell=True)
+        # FIXME: This should be refactored to use the Command module from this framework.
+        # However this would mean a refactor that we can't afford at the moment so we'll
+        # stick with the minimal viable fix.
+
+        # Do not use popen.check_call because it won't redirect stderr to stdout properly
+        # and it can't close stdin which causes issues in certain configurations on Windows.
+        if sys.platform == "win32":
+            # More on this Windows specific fix here: https://bugs.python.org/issue3905
+            p = subprocess.Popen(
+                cmd, shell=True,
```

Comment:
I used an array based approach originally but I felt it made the commands unreadable and harder to edit. Then I realized the inputs were safe therefore using shell=True wouldn't be too bad and then the command strings looked a lot cleaner.